benchmarks/domains/transfer_learning/michalewicz/michalewicz_tl_continuous.py

```diff
@@ -85,18 +86,22 @@ def apply_function(row):
 
 
 def make_initial_data(
-    function: Callable, function_name: str, num_of_points: int
+    function: Callable,
+    function_name: str,
+    num_of_points: int,
+    settings: ConvergenceBenchmarkSettings,
```

Comment:
Unnecessary coupling of objects. Either replace the `settings` argument with a plain `random_seed` or – and that would make it even more general – remove the argument altogether and just wrap the call from the outside, like you did in all other modules

Reply:
Made unified with the rest